packages/benchmark/src/testExecution/testDataImporter.ts

```diff
@@ -0,0 +1,57 @@
+import { AuthenticatedN8nApiClient } from '@/n8nApiClient/authenticatedN8nApiClient';
+import { Workflow } from '@/n8nApiClient/n8nApiClient.types';
```

Comment:
Linting setup seems incorrect if it's not flagging that this import needs `type`.

Reply:
I'll introduce linting in a separate PR